tools/ovc/openvino/tools/ovc/__init__.py

```diff
@@ -1,4 +1,19 @@
 # Copyright (C) 2018-2024 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
 
-from openvino.tools.ovc.convert import convert_model
\ No newline at end of file
+from openvino.tools.ovc.convert import convert_model
+from openvino.tools.ovc.telemetry_utils import is_optimum, init_mo_telemetry
+
+import importlib.metadata as importlib_metadata
+
+try:
+    optimum_version = importlib_metadata.version("optimum")
```

Comment:
this is not reliable check, it can be optimum package but without openvino backend support for example.
Better to check
```suggestion
    optimum_version = importlib_metadata.version("optimum-intel")
```